### PR TITLE
Release v0.31.0

### PR DESCRIPTION
**Pull request by bexsoft**

### CODE REVIEW

**Review by prakashsvmx — APPROVED** [review]

**Review by kaankabalak — DISMISSED** [review]

**Review by dvaldivia — DISMISSED** [review]

**Review by cesnietor — CHANGES_REQUESTED** [review]
@bexsoft  could you please add the changes to the changelog?

**Comment by donatello** [comment]
Please take this in before merging this PR - https://github.com/minio/console/pull/2914

**Review by cesnietor — APPROVED** [review]
LGTM

**Review by dvaldivia — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,24 @@
...
+
+*Breaking Changes:*
+
+- *Removed support for Standalone Deployments*
+
+Features:
+
+- Added React Virtualize to Object Manager
+- Migrated Audit Logs module to mds
+- Migrated Policies module to mds
```
[the remark this answers is not in the document]
Comment (cesnietor) [reply CHANGELOG.md:15]: proposal:
- Updated way files displayed in uploading component
- Updated Audit Logs and Policies components

[the remark this answers is not in the document]
Comment (bexsoft) [reply CHANGELOG.md]: Fixed

**Review by cesnietor — APPROVED** [review]
LGTM

**Pull request merged**